build/targets/KoreBuild.SolutionBuild.targets

```diff
@@ -290,6 +290,8 @@ Runs the VSTest on all projects in the ProjectsToTest itemgroup.
 
   <PropertyGroup>
     <VSTestLogger Condition=" '$(VSTestLogger)' == '' AND '$(TEAMCITY_VERSION)' != '' ">trx</VSTestLogger>
+    <!-- ensure AppVeyor's auto test reporter remains enabled -->
+    <VSTestAutoReporters Condition="'$(APPVEYOR_API_URL)' != ''">true</VSTestAutoReporters>
```

Comment:
Does something set `$(VSTestLogger)` when on AppVeyor? If not, the `$(VSTestLogger)` condition may be enough -- no need for `$(VSTestAutoReporters)`.